packages/acs-ui-common/eslint.config.js

```diff
@@ -0,0 +1,134 @@
+const { defineConfig, globalIgnores } = require('eslint/config');
+
```

Comment:
I'm wondering if it's possible to create a common eslint config and reuse it in the packlets?